doc/source/notebooks/advanced_group_stressors.ipynb

```diff
@@ -34,7 +34,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "wiod_folder = '/tmp/mrios/WIOD2013'"
+    "wiod_folder = \"/tmp/mrios/WIOD2013\""
```

Comment:
This happens on multiple places - is that something black does? Or did you do that manually?